hwstats/backend.py

```diff
@@ -90,3 +90,16 @@ def query_memory_percent_with_time(pid_hash: str, session: Session) -> list[tupl
         .all()
     )
     return result
+
+
+def query_Disk_percent_with_time(pid_hash: str, session: Session) -> list[tuple[datetime, float]]:
+    result = (
+        session.query(
+            Disk.measurementTime.label("timestamp"),
+            Disk.diskRead.label("data"),
```

Comment:
@SiefSallam both the diskRead and diskRight results are being given the same label of "data". Since this query is getting two pieces of information, we'll need to give them unique labels like `readData` and `writeData` or something like that